uwblueprint/community-fridge-kw
# feat: Create scheduling process skeleton

## Notion ticket link
Initial Scheduling Structure - https://www.notion.so/uwblueprintexecs/Initial-Scheduling-Structure-83f15a6202fe476a99524954a8b24fac



## Implementation description
* Created skeleton for scheduling a donation process based on lo-fi Figma designs.
* Consists of the following pages:

1. Get started with scheduling a donation
2. Select date and time
3. Donation information
4. Volunteer information
5. Confirm details
6. Thank you page

Note 1-5 are using the useStep functionality in `/schedule` route while 6. Thank You page is its own route `/schedule/confirmation`


https://user-images.githubusercontent.com/19617248/140828950-7edf0945-d734-4c21-bfa7-cfb9d37cdca4.mp4




## Steps to test
1. Make sure it compiles with no errors (warnings will be resolved as pages are filled in)
2. Go to home page and click on 'Schedule a drop off'
3. Click through steps using the 'Next' button and 'Submit' button and make sure all necessary pages exist. (Components will be filled in)


## What should reviewers focus on?
* 


## Checklist
